features/supplier/supplier_account.feature

```diff
@@ -28,6 +29,25 @@ Scenario: Supplier user can edit supplier information
     | Supplier summary          | All fresh                       |
 
 
+@skip-preview @skip-staging
```

Comment:
This one should only be skip-staging I think.

Reply:
it needs to be @skip-preview first, so when I merge it, the tests won't fail with old version of the code. Then I merge new code, I merge new tests without @skip-preview tag